.travis-ghc.sh

```diff
@@ -0,0 +1,6 @@
+#/usr/bin/env bash
+set -ev
+
+for package in $(cat sources.txt); do
```

Comment:
Using the `SOURCES` array from `scripts/common/lib.sh` has the advantage of making the script runnable from any dir (and also is more consistent with what we do elsewhere).